pmg/models/resources.py

```diff
@@ -192,6 +192,10 @@ def __unicode__(self):
     def list(cls):
         return cls.query.order_by(desc(cls.year))
 
+    @classmethod
+    def committee_bill(cls):
```

Comment:
I think this should be plural - it returns all the matching bills, right?

Reply:
They get combined in the view template , not in the API

Reply:
Committee has its own api endpoint.